### PR TITLE
Fix pre-commit linting issues

### PR DESCRIPTION
**Pull request by amolkahat**
This patch fixes pre-commit linting issues.

### CODE REVIEW

**Comment by amolkahat** [comment]
Example output:
```
akahat@jatayu tmp $ ansible-playbook check_spaces.yaml
PLAY [localhost] *****************************************************************************************

TASK [debug] *********************************************************************************************
ok: [localhost] => {
    "hosts[0]": "127.0.0.1   localhost.localdomain localhost.localdomain1 localhost.localdomain2 localhost.localdomain3 localhost.localdomain4"
}

PLAY RECAP ***********************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   


akahat@jatayu tmp $ cat check_spaces.yaml 
---
- hosts: localhost
  gather_facts: false
  vars:
    hosts:
      - "127.0.0.1   localhost.localdomain localhost.localdomain1 \
        localhost.localdomain2 localhost.localdomain3 localhost.localdomain4"

  tasks:
    - name: "debug"
      debug:
        var: hosts[0]

akahat@jatayu tmp 
```

**Comment by Sandeepyadav93** [comment]
Before this change, failing for following, this change looks good to me.
~~~
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 2

devsetup/edpm/edpm-play.yaml:216:151: [warning] line too long (213 > 150 characters) (line-length)

Ansible-lint.............................................................Failed
- hook id: ansible-lint
- duration: 1.63s
- exit code: 2

INFO     Set ANSIBLE_LIBRARY=/home/sandeepyadav/.cache/ansible-compat/8ef565/modules:/home/sandeepyadav/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/sandeepyadav/.cache/ansible-compat/8ef565/collections:/home/sandeepyadav/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/home/sandeepyadav/.cache/ansible-compat/8ef565/roles:/home/sandeepyadav/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
INFO     Discovered files to lint using: git -c safe.directory=/tmp/install_yamls ls-files --cached --others --exclude-standard -z
INFO     Excluded removed files using: git -c safe.directory=/tmp/install_yamls ls-files --deleted -z
INFO     Discovered files to lint using: git -c safe.directory=/tmp/install_yamls ls-files --cached --others --exclude-standard -z
INFO     Excluded removed files using: git -c safe.directory=/tmp/install_yamls ls-files --deleted -z
WARNING  Listing 1 violation(s) that are fatal
devsetup/edpm/edpm-play.yaml:216: yaml[line-length]: Line too long (213 > 150 characters)
You can skip specific rules or tags by adding them to your configuration file:
# .config/ansible-lint.yml
warn_list:  # or 'skip_list' to silence them completely
  - yaml[line-length]  # Violations reported by yamllint.

                Rule Violation Summary                
 count tag               profile rule associated tags 
     1 yaml[line-length] basic   formatting, yaml     

Failed after min profile: 1 failure(s), 0 warning(s) on 40 files.

bashate..................................................................Failed
- hook id: bashate
- exit code: 1

scripts/gen-nncp.sh:109:1: E003 Indent not multiple of 4
1 bashate error(s) found

pre-commit hook(s) made changes.
If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
To run `pre-commit` as part of git workflow, use `pre-commit install`.
All changes made by hooks:
~~~

File: devsetup/edpm/edpm-play.yaml [hunk 1]
```diff
@@ -213,7 +213,8 @@ spec:
                 undercloud_hosts_entries: []
                 # edpm_hosts_entries role
                 extra_hosts_entries:
-                - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
+                - "172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc \
+                  nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc"
```
Comment (stuggi) [inline devsetup/edpm/edpm-play.yaml:217]: could also use the folded block scalar '>' which will fold newlines to spaces

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**